package.json

```diff
@@ -25,7 +25,7 @@
     "clean": "rm -rf node_modules/.cache && find . -name *.tsbuildinfo -type f -delete",
     "flow-types": "scripts/flow-types.sh",
     "postinstall": "npx yarn-deduplicate && yarn --ignore-scripts",
-    "lint": "eslint --config .eslintrc.js --ext tsx,ts ./packages/**/src ./examples",
```

Comment:
Are we removing `./packages/**/src ./examples` because it's already included as part of `.`?

Reply:
Yep, that's right. I added in some excludes so it doesn't pick up too much either.